capsules/src/FXOS8700CQ.rs

```diff
@@ -0,0 +1,130 @@
+use core::cell::Cell;
```

Comment:
Add a descriptive comment at the top: what is this capsule for? what does it do? How do I initialize and use it? (use `//!` style comments)